.circleci/gradle.properties

```diff
@@ -1,5 +1,5 @@
-org.gradle.jvmargs=-Xmx3072M -XX:+UseParallelGC -Dkotlin.daemon.jvm.options\="-Xmx3072M"
-org.gradle.workers.max=2
+org.gradle.jvmargs=-Xmx5120M -XX:+UseParallelGC -Dkotlin.daemon.jvm.options\="-Xmx5120M"
```

Comment:
I would expect 4608M to match the earlier ratio. Where did you get the 5120 from?

Reply:
We seemed to be getting along fine on avoiding the JVM processes exceeding the container memory with the 1G headroom, so I kept that consistent.